plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2847,6 +2847,47 @@ public void testBasicTableStatistics()
         dropTable(tableName);
     }
 
+    @Test
+    public void testBasicAnalyze()
```

Comment:
nit: you can also test if stats go away if you explicitly drop them.

Reply:
This is covered by `io.trino.plugin.iceberg.TestIcebergAnalyze#testDropExtendedStats`